macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+class TestColorViewController: NSViewController {
+	override func loadView() {
+		let containerView = NSView()
+		let scrollView = NSScrollView()
+		scrollView.translatesAutoresizingMaskIntoConstraints = false
+		scrollView.hasVerticalScroller = true
+		containerView.addSubview(scrollView)
+		
+		let colorsStackView = NSStackView()
+		colorsStackView.translatesAutoresizingMaskIntoConstraints = false
+		colorsStackView.orientation = .vertical
+		colorsStackView.alignment = .leading
+		
+		for color in Colors.Palette.allCases {
+			let colorView = ColorRectView(color: color.color)
+			colorView.invalidateIntrinsicContentSize()
```

Comment:
should not be needed

Reply:
I'm creating a rect view and a text view for each color in the palette. What's the alternative way to achieve that?

Reply:
Sorry the highlight was off - we do not need invalidateIntrinsicContentSize. The intrinsicContentSize will always be queried on initial layout and only needs to be invalidated if something about the object changes later that could change its intrinsicContentSize